visualization/app/codeCharta/ui/metricChooser/metricChooser.po.ts

```diff
@@ -1,4 +1,19 @@
 export class MetricChooserPageObject {
+	public async openHeightMetricChooser() {
+		await expect(page).toClick("height-metric-chooser-component md-select", { timeout: 3000 })
+		await page.waitForSelector(".md-select-menu-container.ribbonBarDropdown.md-active")
+	}
+
+	public async clickOnHeightMetricSearch() {
+		await expect(page).toClick("md-backdrop", { button: "middle", timeout: 3000 })
```

Comment:
Why the middle click?

Reply:
It's to unfocus the searchfield. When we first open the MetricChooser the searchfield is automatically focused. So I wanted to unfocus it, by making a middle click, then clicking the searchfield again and check if it got closed.

Reply:
But isn't the search field focused automatically when you click on the metricChooser?
Why do we want to undo this behavior in our e2e test?